ImageFolder/images.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/ImageFolder/images.css">
```

Comment:
image.css is not inside ImageFolder...it is inside of components folder. Change the link. Except this everything looks good!

Reply:
Actually that folder is imported . It's not part of the component library . I will delete it . Thanks!